config/locales/views/groups/en.yml

```diff
@@ -35,6 +35,7 @@ en:
         already_pending: 'Could not invite %{user_name}: this student already has a pending invitation for this group.'
         extension_exists: You cannot modify a group with an extension to the due date. Please contact your instructor if you wish to make changes.
         group_max_reached: 'Could not invite %{user_name}: group maximum has been reached.'
+        inactive_grader: Cannot add '%{user_names}' to group/s because these graders are inactive.
```

Comment:
This message should be displayed when trying to assign graders to groups _or_ criteria. So a more general message would be "Cannot assign inactive grader(s) '%{user_names}'.